### PR TITLE
fix(curriculum) - Add class box in json apis and ajax

### LINKED ISSUE #37713: .box is not used in JSON APIs and Ajax challenges
The css exists, but the class is not applied.  Either the css should be removed or the 'box' class applied.

https://github.com/freeCodeCamp/freeCodeCamp/pull/37069#discussion_r343782958

### PR DESCRIPTION
**Pull request by IMHOJEONG**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #37713

### CODE REVIEW

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-xmlhttprequest-method.english.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ tests:
   }
 </style>
 <h1>Cat Photo Finder</h1>
-<p class="message">
+<p class="message box">
```
Comment (moT01) [inline curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-xmlhttprequest-method.english.md]: Here, you added the class to the solution, you should also add the class to the seed code.

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.english.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ tests:
   }
 </style>
 <h1>Cat Photo Finder</h1>
-<p id="message">
+<p id="message box">
```
Comment (moT01) [inline curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.english.md]: This challenge has a id of message, you added `box` as part of the id. It needs to be a class. This is the only challenge where `message` is the id and not the class. For now, I would just add `box` as a class, but we may want to go through and change this to align with the rest of the challenges. Or, if you want to take a look and maybe rewrite a little, go for it.

**Review by moT01 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-xmlhttprequest-method.english.md (hunk 1 above)
Comment (IMHOJEONG) [reply curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-xmlhttprequest-method.english.md]: Oh, That`s my fault, Thank you for telling me

File: curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.english.md (hunk 2 above)
Comment (IMHOJEONG) [reply curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.english.md]: Yeah, You`re right. I'll fix it and upload it again

Comment (moT01) [reply curriculum/challenges/english/04-data-visualization/json-apis-and-ajax/get-json-with-the-javascript-fetch-method.english.md]: This file was removed from the PR

**Comment by moT01** [comment]
There's still an unresolved suggestion that should be fixed here.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
So there is.  Sorry, @moT01, I didn't read carefully enough.  @IMHOJEONG could you add `class="box"` to get-json-with-the-javascript-fetch-method.english.md?

Also, the solutions should all have the changes (i.e. box), since they're supposed to be complete solutions.  i.e. what you'd get by starting from the seed and then making changes to pass the challenge.

**Comment by moT01** [comment]
If I recall, none of the challenges had solutions except the one in my first suggestion. So the full solutions would need to be added.

**Comment by ojeytonwilliams** [comment]
Ah, I saw the one with a solution and didn't check the rest.  Fair enough, no need to add those in this PR.

**Comment by IMHOJEONG** [comment]
I added class="box" to get-json-with-the-javascript-fetch-method.english.md. What else can I do?

**Review by moT01 — APPROVED** [review]
Not at the moment @IMHOJEONG - thanks for fixing these.

LGTM 🎉

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@ojeytonwilliams If you are OK with everything, you can approve this PR and merge it.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Comment by ojeytonwilliams** [comment]
@IMHOJEONG Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**